docs/compatible_microcontrollers.md

```diff
@@ -62,6 +62,55 @@ You can also use any ARM chip with USB that [ChibiOS](https://www.chibios.org) s
  * [MK66FX1M0](https://www.nxp.com/products/processors-and-microcontrollers/arm-microcontrollers/general-purpose-mcus/k-series-cortex-m4/k6x-ethernet/kinetis-k66-180-mhz-dual-high-speed-full-speed-usbs-2mb-flash-microcontrollers-mcus-based-on-arm-cortex-m4-core:K66_180)
    * PJRC Teensy 3.6
 
+### Raspberry PI RP2040
```

Comment:
The changes here are far too verbose and should be migrated to a separate page.

Reply:
Done. See updated commit 5204c8b159bed75f6a8d2430aa9cdfa4a77159fc